docs/services/hpc-console.md

```diff
@@ -0,0 +1,53 @@
+[](){#hpc-console}
+# HPC Console
+
+HCP Console is a web application designed to provide essential functionalities for interacting with [FirecREST API v2 backend][ref-firecrest].
+It provides users with a modern web-based environment to interact with HPC resources such as clusters, job schedulers, and filesystems.
+
+## Features
+
+- A modern web interface built on FirecREST API functionalities.
+- A dashboard offering an overview of configured clusters and their statuses.
+- A dedicated view for creating and monitoring jobs.
+- A file navigator with basic file management functionalities, including upload and download features.
+
+A description of the views and functionalities can be found [here](https://eth-cscs.github.io/firecrest-ui/documentation/).
+
+## Views overview
+
+- Dashboard – shows cluster availability and basic system status.
+- Jobs – allows creating, submitting, and monitoring jobs. Users can provide job name, script, and resource requirements.
+- File Navigator – supports browsing directories, uploading, downloading, and deleting files. Editing files inline is not supported.
+
+## Supported version
+
+Starting early 2025, CSCS has introduced a new version of the API: [FirecREST version 2](https://eth-cscs.github.io/firecrest-v2).
+
+This web interface is available and runs on the FirecREST API v2 backend, FirecREST API v1 is not supported.
+
+## Alps platforms
+
+HPC console is currently available on three [Alps platforms][ref-alps-platforms].
+
+| Platform       | URL               | Clusters                                                                 |
+|----------------|----------------------|--------------------------------------------------------------------------|
+| HPC Platform   | [https://my.hpcp.cscs.ch/](https://my.hpcp.cscs.ch/) | [Daint](../clusters/daint), [Eiger](../clusters/eiger)             |
+| ML Platform    | [https://my.mlp.cscs.ch/](https://my.mlp.cscs.ch/)  | [Bristen](../clusters/bristen), [Clariden](../clusters/clariden)  |
+| CW Platform    | [https://my.cwp.cscs.ch/](https://my.cwp.cscs.ch/)  | [Santis](../clusters/santis)  |
+
+## Access and prerequisites
+
+- Access requires a valid CSCS account and authentication through the institutional login system.
+- Supported browsers: recent versions of Chrome, Firefox, and Safari are recommended.
+- Sessions automatically expire after a period of inactivity.
+
+## Feedback
+
+We welcome your feedback to help us improve HPC Console.
+If you encounter issues or have suggestions, please let us know via our [Service Desk](https://jira.cscs.ch/plugins/servlet/desk).
+
+## Further information
+
+* [HPC Console doc](https://eth-cscs.github.io/firecrest-ui/home/)
+* [Github Repo](https://github.com/eth-cscs/firecrest-ui)
+* [FirecREST doc](https://eth-cscs.github.io/firecrest-v2)
```

Comment:
```suggestion
* [HPC Console documentation](https://eth-cscs.github.io/firecrest-ui/home/)
* [Github repository](https://github.com/eth-cscs/firecrest-ui)
* [FirecREST documentation](https://eth-cscs.github.io/firecrest-v2)
```

Reply:
C.f. my other question about firecrest ui vs hpc console: Should the first link really say `HPC Console` or should it say `Firecrest UI`? It's not clear to me if the latter will be renamed, or if HPC Console is just an instance of the latter.